tests/unit/testsuite/Bolt/Boltpay/Block/Checkout/BoltpayTest.php

```diff
@@ -2,17 +2,34 @@
 
 require_once('TestHelper.php');
```

Comment:
Please remove this

Reply:
`require_once('TestHelper.php');` was not added in this pull request and would fall under the category of an unrelated change, which we try very hard not to do.  I will not add that here, but will create a separate PR to remove this if I find that it is unnecessary.  You may also do the same.